modules/scanning-images.adoc

```diff
@@ -172,6 +172,29 @@ Scanner can check for vulnerabilities in dependencies for the following programm
 |===
 . Only supported in Scanner V4.
 
+[id="supported-layer-compression-formats_{context}"]
+== Supported layer compression formats
+
+Container image layers are .tar file archives that might be compressed or uncompressed. StackRox Scanner and Scanner V4 support different formats as shown in the following table:
```

Comment:
```suggestion
Container image layers are `.tar` file archives that might be compressed or uncompressed. StackRox Scanner and Scanner V4 support different formats as shown in the following table:
```
- File names and file extensions should be in monospace font, per IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element